tests/foreman/cli/test_remoteexecution.py

```diff
@@ -840,22 +839,11 @@ def test_positive_time_expressions(self, rex_contenthost, target_sat):
                 '@hourly',
                 f'{(datetime.utcnow() + timedelta(hours=1)).strftime("%Y/%m/%d %H")}:00:00',
             ],
-            # 23 mins after every other hour
-            [
-                '23 0-23/2 * * *',
-                f'{today.strftime("%Y/%m/%d")} '
-                f'{(str(hour if hour % 2 == 0 else hour + 1)).rjust(2, "0")}:23:00',
-            ],
```

Comment:
Could you add some description/comments about this removals?

Reply:
the next run calculation tends to be unstable in these cases, I think mainly because the expected value is calculated when building the list, but the jobs are run a bit later in the loop, so if the hour changes meanwhile the assertions won't match.